pkg/console/starter/starter.go

```diff
@@ -86,7 +89,12 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		return err
 	}
 
-	const resync = 10 * time.Minute
+	managedClusterClient, err := clusterclient.NewForConfig(controllerContext.KubeConfig)
+	if err != nil {
+		return err
+	}
+
+	const resync = 30 * time.Second
```

Comment:
why ?

Reply:
Just forgot to change this back to the original value.